src/Functions/throwIf.cpp

```diff
@@ -172,7 +172,31 @@ class FunctionThrowIf : public IFunction
 
 REGISTER_FUNCTION(ThrowIf)
 {
-    factory.registerFunction<FunctionThrowIf>();
+    FunctionDocumentation::Description description = R"(
+Throw an exception if argument x is true.
```

Comment:
\+ `To use the error_code argument, configuration parameter allow_custom_error_code_in_throwif must be enabled.`